src/dag-stream.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+const Writable = require('stream').Writable;
+const util = require('util');
+const Batch = require('./dag-service-batch')
+
+exports = module.exports.BatchStream = BatchStream
```

Comment:
It would be cleaner if

`exports = module.exports`

and then

`exports.BatchStream = (ds, max, options) { ...`